packages/jest-runner/src/types.ts

```diff
@@ -46,6 +50,7 @@ export type TestFramework = (
 ) => Promise<TestResult>;
 
 export type TestRunnerOptions = {
+  onTestProgress?: (test: Test, progress: TestProgress) => void;
```

Comment:
I wanted to add this alongside onStart, onResult, etc in jest-runner. But doing so either would have to go after the options object, or would be a breaking change. I feel like I picked the best among bad choices, but I am open to other ideas here.

I marked it optional, but its only undefined if the runner is running against an older version of @jest/core. What are the api guarantees here? Is it safe to make this required?

Reply:
Optional for now seems fine. We can make more changes to the interface for Jest 26 (such as having it next to `onStart` etc)

Reply:
Can we leave a comment somewhere in the code so it's harder to forget?

Reply:
Or open an issue and I'll add it to the milestone